shared-libs/lineage/package-lock.json

```diff
@@ -5,11 +5,605 @@
   "requires": true,
   "dependencies": {
     "@medic/contact-types-utils": {
-      "version": "file:../contact-types-utils"
+      "version": "file:../contact-types-utils",
```

Comment:
Same here, you might need to leave these medic dependencies as it is to avoid errors

Reply:
reverted in 91e83b5d7